projects/imgix-angular/src/public-api.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Public API Surface of imgix-angular
+ */
+
+export * from './lib/imgix-angular.module';
+export * from './lib/ix-img.component';
+export * from './lib/ix-source.component';
```

Comment:
Should this also export `./lib/ix-picture.component.ts`?

Reply:
Sure, why not 🤷